### ISSUE TITLE
Add incoming connections list to the admin resource edit page

### ISSUE THREAD

**Issue by mworrell** [zotonic/zotonic#1195]
Proposal: add tabs to the connection list: incoming & outgoing. 

Where we:

 * limit the number of connections shown (to prevent problems with eg. keywords)
 * order by seq for outgoing, date descending for incoming

The incoming tab can be lazy loaded, so that we don't need to do extra queries.

Maybe we can also prune the number of outgoing connections - too long lists don't make any sense anyway. And then place a link to the connection overview page for the complete overview.

/cc @ddeboer @DorienD @ArthurClemens

**Comment by mmzeeman** [comment]
Sounds nice. Do you want to mix all predicates and visualise the connection type in another way?

**Comment by mworrell** [comment]
I was thinking of a list:

 * Grouped by predicate
 * Max N per predicate, with a *more…* link to the connection overview page
 * Per group ordered by descending creation date

**Comment by mmzeeman** [comment]
BTW: What are the use cases which needs to be covered with the incoming connections list?

**Comment by arjan** [comment]
Agreed. Plus, I can imagine it is confusing that incoming connections are not sortable while outgoing connections are. How are we going to explain this?

**Comment by mworrell** [comment]
We see in projects more and more that both directions are important and should be easy to view.
For the incoming we can add a text "most recently added connections to this page"

**Comment by mworrell** [comment]
This will also make it easier for use cases as:

 * Add this page to a collection (or remove it from a collection)
 * Managing family relationships (thinking of Jewish Monument)
